constants/constants.go

```diff
@@ -0,0 +1,7 @@
+package constants
+
+const (
+	Filename           = "hosts.env"
+	EditByNameTypeURL  = "https://api.porkbun.com/api/json/v3/dns/editByNameType/"
+	RetrieveByNameType = "https://api.porkbun.com/api/json/v3/dns/retrieveByNameType/"
```

Comment:
Should we have these in an env file? If any of the providers change their URLs we'll have to make a change in the repo, but if we have these in an env file, it'll just be a matter of updating your local env file.

Reply:
Agreed, maybe even a separate file in the future or something if this gets too large. 👍